composer.json

```diff
@@ -20,12 +20,12 @@
     },
     "require": {
         "php": "^8.2",
-        "typo3/cms-core": "13.2.* || ^12.4",
-        "typo3fluid/fluid": "<=2.14.1"
+        "typo3/cms-core": "^13.3 || ^12.4",
+        "typo3fluid/fluid": "^4.0 || ^2"
```

Comment:
I would specify a minor version for Fluid 2 here if you depend on some of the new features: `^4.0 || ^2.15`

Reply:
Oh good question